lib/xypn_seo/seo_evaluation/advisor_site_evaluation.rb

```diff
@@ -3,41 +3,40 @@
 module XYPNSEO
   module SEOEvaluation
     class AdvisorSiteEvaluation
-
       def call
         evaluate_all_websites
       end
 
       private
 
       def evaluate_all_websites
-        Website.all.each do | website |
-          desktop_api_call = craft_api_call(website.url, "desktop")
-          mobile_api_call = craft_api_call(website.url, "mobile")
+        Website.all.each do |website|
+          desktop_api_call = craft_api_call(website.url, 'desktop')
+          mobile_api_call = craft_api_call(website.url, 'mobile')
           website.desktop_score = obtain_score(desktop_api_call)
           website.mobile_score = obtain_score(mobile_api_call)
           website.save!
         end
       end
 
       def obtain_score(api_url)
-        begin
-          api_response = RestClient.get(api_url)
-          response_as_json = JSON.parse(api_response.body)
-          response_as_json['score']
-        rescue RestClient::BadRequest => e
-          Rails.logger.error(e)
-          return 0 # Website score columns placeholder, required for correct scope querying
-        rescue RestClient::InternalServerError => e
-          Rails.logger.error(e)
-          return 0 
-        end
+        api_response = RestClient.get(api_url)
+        response_as_json = JSON.parse(api_response.body)
+        response_as_json['score']
+      rescue RestClient::BadRequest => e
+        Rails.logger.error(e)
+        return 0
+        # Website score columns placeholder, needed for scope querying
+      rescue RestClient::InternalServerError => e
+        Rails.logger.error(e)
+        return 0
       end
 
       def craft_api_call(site_url, strategy)
```

Comment:
Hmm...I'd be OK with violating the rubocop's rules here.  They're not gospel.  They guidance.

Alternatively, does this approach work?  It's a little more robust than just string interpolation.
```
params = {
  url: #{site_url},
  key: #{ENV['PAGESPEED_KEY']},
  prettyprint: false,
  strategy: #{strategy}
}
uri = URI::HTTP.build(host: "https://www.googleapis.com/pagespeedonline/v1/runPagespeed", query: params.to_query)

Play around with this if you have time, if not, ship what you've got.  No sense is spending a ton of time on this issue.  It's very minor.